source/_integrations/reolink.markdown

```diff
@@ -621,6 +621,81 @@ Prerequisites:
 
 - Turn on (outdoor) lights near the camera to improve image clarity at night once the camera detects a person, vehicle, or animal.
 - Turn off notifications and recording when you get home (based on, e.g., geofencing) and turn it back on when you leave home.
+- Auto-pause rich notifications for x time
+
+{% details "Auto-pause rich notifications tutorial" icon="mdi:cursor-hand" %}
+
+**Goal**: At the end of this tutorial, you will have a drop-down on your dashboard with different time choices to pause your notifications. When the time is up, the notifications will become active again. It will look like this:
+
+![Overview of end result](/images/integrations/reolink/auto_pause__overview.png)
+
+1. First, create the dropdown from **Settings** > **Devices & services** > **Helpers** > **+ Create Helper** > **Dropdown**. 
+   - Decide how many time delay choices you want. 
+   - Add them all to the dropdown like below. 
+   - Your first entry needs to be "Notifications active" (or simular phrasing) for when the notifications are turned on. 
+   - The time amounts can be any amount and you can have as many as you want to code.
```

Comment:
```suggestion
   - You can define as many time options as you want. And you can define any time interval you like, for example, 22 minutes, 2 hours.
```
I struggled a bit at first with the "Time amount". Maybe this would be an option? Feel free to rephrase or ignore if you think it isn't accurate.

Reply:
No I think this is much better phrased!

Reply:
I am not so good in writing texts/docs. But it comes with the programming....

Reply:
haha, I'm not good with programming. Making things is hard. Nitpicking is always easy. Thank you for making this happen. :rocket: